app/controllers/api/entities/test_question.rb

```diff
@@ -1,8 +1,10 @@
 module API
   module Entities
     class TestQuestion < Grape::Entity
-      expose :question, using: API::Entities::Question
-      expose :answers
+      expose :id
+      expose :question, using: API::Entities::QuestionWithAnswers
+      expose :answers, as: :chosen_answers, with: API::Entities::Answer
+      expose :correct, if: ->(object, _options){object.test.status != "doing"}
```

Comment:
```suggestion
      expose :correct, if: ->(object, _options){!object.test.doing?}
```
ri đi emmmm
hạn chế so sánh biến cố định thế này mà còn string thì hok nên hì